uiowa/uiowa01
# Feature: Add HDS search / Remove additional header alignment options

Work on issues #312 and #306.

# How to test

1. `drush @theming.local si collegiate`
2. Verify that site install has 'site name left, menu left' 
3. Enable uiowa_book module
4. Go to a-z list page and verify HDS styles are applied: https://theming.uiowa.local.site/a-z
5. Add views exposed search block to page with layout builder enabled and verify that styles are there
6. Let me know if you have any feedback on https://github.com/uiowa/uiowa01/pull/315#issuecomment-572709647
